app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>All Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+  <h3>Name: </h3><%= contestant.name %>
+	<h3>Projects: </h3><% contestant.projects.each do |project| %>
+		<%= project.name %>
+		<% end %>
+<% end %>
```

Comment:
Please don't forget that at the end of the day this is an HTML file and therefore all of your rendered erb tags , `<%= %>`, need to be enclosed in an HTML element.

```erb
<% @contestants.each do |contestant| %>
  <h3>Name: <%= contestant.name %></h3>
  <h3>Projects:</h3>
  <% contestant.projects.each do |project| %>
    <p><%= project.name %></p>
  <% end %>
<% end %>

```